src/main/plugins/org.dita.html5/xsl/hi-d.xsl

```diff
@@ -63,15 +63,15 @@ See the accompanying LICENSE file for applicable license.
   </xsl:template>
 
   <xsl:template match="*[contains(@class,' hi-d/line-through ')]" name="topic.hi-d.line-through">
-    <span style="text-decoration:line-through">
+    <span>
```

Comment:
I'm a little conflicted on this, just because the style is really part of the element itself, so it sort of makes sense to have it in the HTML rather than the CSS. I'm thinking of a case like my own, where we don't use the toolkit's CSS at all, so updating to this version of OT would look like a regression.

Generally ambivalent though because I also don't like having hard coded style attributes; one way to counter the concern might be to list this among the "migrating to 3.6" instructions?

Reply:
I can see the line-through/overline cases either way as well, and I understand this requires extra work in cases where the default CSS is not used.

The main issue I see is that the current code makes it hard to override the default presentation for these elements _at all_, since inline `@style` attributes take precedence over any CSS rules in external style sheets.

So even if a frontend developer wanted to replace overlining with something like animated top borders (as many do with link _underlining_), the hard-coded overlines would still be visible.

Either way, these changes should certainly be documented in the 3.6 migration info.

Reply:
I'd prefer to move this to CSS and not hard-code it in HTML.